src/content/en/updates/2017/02/improved-add-to-home-screen.md

```diff
@@ -74,21 +74,75 @@ site.
+instead of Chrome openning with your PWA running.
+
+When you install a Progressive Web App, we look at your Web App Manifest and
+other meta-data and create the APK that is installed on to the user's device. In
+that APK we define an [Android Intent
+Filter](https://developer.android.com/guide/components/intents-filters.html)
+that defines when your web application should be opened. For example, to
+open the [https://airhorner.com](https://airhorner.com/) app whenever that link
+is clicked, Chrome would create the following `<intent-filter>`.
+
+    <intent-filter>
+      <action android:name="android.intent.action.VIEW" />
+      <category android:name="android.intent.category.DEFAULT" />
+      <category android:name="android.intent.category.BROWSABLE" />
+      <data
+        android:scheme="https"
+        android:host="airhorner.com"
+        android:pathPrefix="/" />
+    </intent-filter>
+
+This is incredibly powerful, but not very flexible. This `<intent-filter>`
+simply says when a link that is clicked or intercepted in Android for the entire
+domain of `https://airhorner.com/` open the App.
+
+But what if you don't want your PWA to open for all paths on your domain? That
+is where the `scope` [Web App
+manifest](https://www.w3.org/TR/appmanifest/#scope-member) property comes in to
+play. The `scope` defines a sub path (essentially it alters the
+`android:pathPrefix` attribute in the APK's `AndroidManifest.xml`) defaults to
+the origin. You can set it to an path that is relative to your origin and
+subsequently when a user navigates to a URL contained by the scope your
+installed Progressive Web App will be open.
 
 Note: directly navigating to your site from the address bar will work exactly 
 the same as it does for native apps that have an intent filter, Chrome assumes 
 that the user intended to visit the site and will open the site.
 
+A specific example, is that of a large web site, that has many different content
+sections (/tech-news/ , /celebs/, /business/) and the they would like to have a
+PWA just for their '/tech-news/' section, they would set the scope to be `scope:
+'/tech-news/',` in the manifest to ensure that only links to that part of the
+site open up the Progressive Web App.
+
+If you were to define a `scope` in your Web App Manifest, you would see an
+updated `pathPrefix` in the `<intent-filter>` in your apps
+`AndroidManifest.xml`.
+
+    <intent-filter>
+      <action android:name="android.intent.action.VIEW" />
+      <category android:name="android.intent.category.DEFAULT" />
+      <category android:name="android.intent.category.BROWSABLE" />
+      <data
+        android:scheme="https"
+        android:host="kinlansnewssite.com"
+        android:pathPrefix="/tech-news/" />
+    </intent-filter>
+
+Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
+your Service Worker. It is possible for one installed Web Application to be
+controlled by several different Service Workers.
```

Comment:
It seems like this is something that should go in the first paragraph that mentions scope.